Source/Scene/Expression.js

```diff
@@ -43,6 +43,19 @@ define([
         }
     };
 
+    var unaryFunctions = {
+        abs : Math.abs,
+        sqrt : Math.sqrt,
+        cos : Math.cos,
+        sin : Math.sin,
+        tan : Math.tan,
+        acos : Math.acos,
+        asin : Math.asin,
+        atan : Math.atan,
+        radians : CesiumMath.toRadians(),
+        degrees : CesiumMath.toDegrees()
```

Comment:
Hm.. just to get something working quickly you can edit the electron version in package.json to a known working version and rerun npm install.